src/index.js

```diff
@@ -106,25 +110,31 @@ class IronStorage {
     async set (key, value) {
         key = `${DEFAULT_KEY_PREFIX}${key}`
 
-        for (const store of this.stores) {
-            // TODO(grun): Run in parallel.
+        let ops = this.stores.map(async store => {
```

Comment:
I'd like to ask you, why did you decide to use the map method instead of for..of here? In general, what I think is that, if we don't really have to return a new array from it, we should use for..of to just iterate through. What do you think?

Reply:
`for...of` would end up running this in series, not parallel. By using map, you're mapping the stores and an array of promises that are running concurrently, then you can await on that array of promises using `Promise.all`

Reply:
Everything you just wrote is completely wrong. Yes the main run loop is single threaded, but if the store supports asynchronously reading/writing a value (say by storing over the nwtwork) then you can absolutely run things concurrently in javascript. And you're understanding of `for...of` vs. `forEach/map` when it comes to async/await is just incorrect. https://stackoverflow.com/questions/37576685/using-async-await-with-a-foreach-loop

Reply:
Regardless of whether you "workerrise" these, the point is for this to be async-ready for any store that supports it. I can easily drop in stores that read/store values over a Rest API and this would support doing those operations concurrently, the current code does not.